Module-6-Assignment/AccountContactsTest.cls

```diff
@@ -0,0 +1,59 @@
+/**
+ * Test class to test the functionality of AccountContacts Class 
+ */
+@isTest
+public class AccountContactsTest 
+{
+	@isTest
+    private static void testWithOneContactId()
+    {
+        Account acc = new Account(Name='Test Account', ContactIds__c = null);
+        insert acc;
+        Contact con = new Contact(LastName='Test contact', AccountId=acc.id);
+        insert con;
+        Test.startTest();
+        String resultantContactIds = AccountContacts.setContacts(con.id);
+        Test.stopTest();
+        System.assertEquals(con.id + ',', resultantContactIds);
+    }
+    
+    @isTest
+    private static void testWithBulkContactIds()
+    {
+        Account acc = new Account(Name='Test Account', ContactIds__c = null);
+        insert acc;
+        List<Contact> contactList = new List<Contact>();
+        for(Integer i = 0; i < 12; i++)
```

Comment:
use a list size of 101

Reply:
Not possible.
Because the maximum size of ContactIds field is 255 characters. It cannot hold that much large amount of data.